presets/cisco/README.md

```diff
@@ -0,0 +1,25 @@
+# Cloudflare presets
+
+This preset directory contains Cisco data sources. Current supported data sources:
+- firepower
+- umbrella_dns
+
+## Firepower
+
+Assumed data format: text
+
+Silver tables:
+    - cisco_firepower
+
+Gold tables:
+    - network_activity
+
+## Umbrella DNS
+
+Assumed data format: text
+
+Silver tables:
+- cisco_firepower
```

Comment:
I dont know what cisco firepower is but seems strange at first glance that DNS goes in there as well.  I assume this is correct?